packages/framework/src/Migrations/Version20210127085903.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20210127085903 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('UPDATE category_translations SET name = NULL WHERE name = \'\'');
```

Comment:
You need to trim even here in SQL like:
```suggestion
        $this->sql('UPDATE category_translations SET name = NULL WHERE TRIM(name) = \'\'');
```